src/main/java/com/catapi/model/Breed.java

```diff
@@ -0,0 +1,65 @@
+package com.catapi.model;
+
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.Id;
+import jakarta.persistence.Table;
+
+@Entity
+@Table(name = "breed")
+public class Breed {
+
+    @Id
+    @Column(name = "id")
+    private long id;
```

Comment:
Add it under id field. Read about it
@GeneratedValue(strategy = GenerationType.IDENTITY)

Reply:
Added and read